pages/docs/adapters/calculix/adapter-calculix-get-adapter.md

```diff
@@ -13,15 +13,19 @@ There are two ways to get the adapter: (a) get a binary package (Ubuntu-only), o
 
 You can download version-specific Ubuntu (Debian) packages from each [adapter release](https://github.com/precice/calculix-adapter/releases/latest). To install, simply open it in your software center.
 
-Alternatively, download & install it from the command line. For Ubuntu 20.04 (focal):
+Alternatively, download & install it from the command line. For Ubuntu 22.04 (jammy):
 
 ```bash
-wget https://github.com/precice/calculix-adapter/releases/download/v{{ site.calculix_adapter_version }}/calculix-precice2_{{ site.calculix_adapter_version }}-1_amd64_focal.deb
-sudo apt install ./calculix-precice2_{{ site.calculix_adapter_version }}-1_amd64_focal.deb
+wget https://github.com/precice/calculix-adapter/releases/download/v{{ site.calculix_adapter_version }}/calculix-precice2_{{ site.calculix_adapter_version }}-1_amd64_jammy.deb
+sudo apt install ./calculix-precice2_{{ site.calculix_adapter_version }}-1_amd64_jammy.deb
 ```
```

Comment:
I'm starting to wonder if adding a website variable "name of the latest Ubuntu" would be interesting, or an unmaintainable overkill.

Reply:
Replied above